cas/ctg_store.c

```diff
@@ -500,7 +595,7 @@ static int ctg_state_create(struct m0_be_seg     *seg,
 			    struct m0_cas_state **state)
 {
 	struct m0_cas_state *out;
-        struct m0_be_btree  *bt;
+        struct m0_btree     *bt;
```

Comment:
Start with tab instead of spaces.
In the lines, you have not touched it is ok but where you have made some change, you need to have this.